allennlp/commands/train.py

```diff
@@ -293,14 +293,15 @@ def train_model(params: Params,
     for name, parameter in model.named_parameters():
         if any(re.search(regex, name) for regex in no_grad_regexes):
             parameter.requires_grad_(False)
-            nograd_parameter_names.append(name)
-        else:
+        if parameter.requires_grad:
             grad_parameter_names.append(name)
+        else:
+            nograd_parameter_names.append(name)
 
-    logger.info("Following parameters are Frozen  (without gradient):")
+    logger.info("FROZEN PARAMETERS  (WITHOUT GRADIENT):")
```

Comment:
No shouting in the logs please 😄

Reply:
Ohh Cool, will revert it back then. I thought I was supposed to capitalize the headings similar to "CURRENTLY DEFINED PARAMETERS".